doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -5,17 +5,51 @@
 
 ## Table of Contents
 * [5.1 Introduction](#5.1)
-* [5.2 Heading](#5.2)
-* [5.3 Heading](#5.3)
+* [5.2 Methodology](#5.2)
+* [5.3 Protoype VNFs](#5.3)
+* [5.4 Certification Strategy & Vehicle](#5.4)
+* [5.5 Profiles Reference](#5.5)
+* [5.6 Compliance, Verification, and Certification process](#5.6)
+* [5.7 Entry & Exit Criteria](#5.7)
+* [5.8 Frameworks](#5.8)
 
 <a name="5.1"></a>
 ## 5.1 Introduction
 
 > Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
 
 <a name="5.2"></a>
-## 5.2 Heading
+## 5.2 Methodology
 
+- Content to be written.
+- Describe the methodolgy and selection process for VNF C&V Framework requirements
 
 <a name="5.3"></a>
-## 5.3 Heading
+## 5.3 Protoype VNFs
+
+- Introduce concept and purpose for Golden VNFs &/or Prototype VNFs
+
+<a name="5.4"></a>
+## 5.4 Certification Strategy & Vehicle
+
+- Describe the certification process, and strategy based on how requirements will be verfiied.
+
+<a name="5.5"></a>
+## 5.5 Profiles Reference 
+
+- Describe the different profiles used for testing and verification 
+
+<a name="5.6"></a>
+## 5.6 Compliance, Verification, and Certification process
+
+- Describe the certification process
+
+<a name="5.7"></a>
+## 5.7 Entry & Exit Criteria
+
+- Provide a list of entry and exit criteria to be used for test certification
+
+<a name="5.8"></a>
+## 5.8 Frameworks
+
+- Provide a list of frameworks used, and their purpose for VNF certification
```

Comment:
where chapter 2 looks at requirement for NFVI Testing Framework, this will look at requirement of VNF framework. (without specifying any). I would suggest clarifying the text for this section.

Reply:
@rabi-abdel - updated verbiage for section 5.8.  Please review for possible closure to the conversation.